java/impl/cmd-line/src/test/java/co/worklytics/psoxy/ConfigTest.java

```diff
@@ -0,0 +1,33 @@
+package co.worklytics.psoxy;
+
+import org.apache.commons.lang3.NotImplementedException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ConfigTest {
+
+    @Test
+    public void validate_secretReference() {
```

Comment:
I'd change the name, as it is validating GCP secrets type -> `validate_gcp_secretReference`